pandas/plotting/_matplotlib/core.py

```diff
@@ -153,8 +169,8 @@ def __init__(
 
         self.grid = grid
         self.legend = legend
-        self.legend_handles = []
-        self.legend_labels = []
+        self.legend_handles: List = []
```

Comment:
Can you subtype these? legend_labels is probably `List[Label]`; not sure about this one but would be helpful for code reviewers

Reply:
ahh, okay, was not meant to annotate this, but mypy complains, I will add subtypes for those two!